app/build.gradle

```diff
@@ -61,4 +70,23 @@ dependencies {
 
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9")
 
+    //mlkit
+    implementation 'com.google.mlkit:barcode-scanning:17.2.0'
+
+    //it.jsoup
+    implementation "org.jsoup:jsoup:1.14.3"
+
+
+    //androix.compose
+    implementation(platform('androidx.compose:compose-bom:2023.08.00'))
```

Comment:
Du hast keine Compose Elemente in deinem Code, also sollten die dependencies auch nicht mehr dabei sein.